CMakeLists.txt

```diff
@@ -94,6 +74,38 @@ add_subdirectory(platform)
 add_subdirectory(rtos)
 add_subdirectory(targets)
 
+#Verify that the selected application OS profile is supported by the Mbed target
+if(NOT MBED_CONF_RTOS_PRESENT)
```

Comment:
`MBED_CONF_RTOS_PRESENT` is define, not CMake variable - does this work?

Reply:
@rajkan01 How was this tested?

Reply:
It works

Reply:
How was it tested?